Dockerfile

```diff
@@ -23,4 +23,4 @@ RUN cp /go/bin/polly /usr/local/ && chmod u+x /usr/local/polly
 # Start the Polly Service
 ENTRYPOINT polly service start -f
 
-EXPOSE 7979 7980
\ No newline at end of file
+EXPOSE 7978 7979
```

Comment:
I thought that was only on the command line with a -p
otherwise I can just divide on two separate lines.